### ISSUE TITLE
inflation table missing from conda package

### ISSUE THREAD

**Issue by knaaptime** [spatialucr/geosnap#321]
something weird is going on with the anaconda releases. We have [a file](https://github.com/spatialucr/geosnap/blob/master/geosnap/io/bls_inflation.csv) inside the package that stores inflation weights. On the pypi release, that file is included in [the tarball](https://files.pythonhosted.org/packages/3e/e6/ff25796e783a2bcc51ac5666c30b48eba20f3b924e9f6aebc1e04ccec7d9/geosnap-0.8.tar.gz), but in the [anaconda release](https://anaconda.org/conda-forge/geosnap/0.8/download/noarch/geosnap-0.8-pyhd8ed1ab_0.tar.bz2) the file is excluded. I cant figure out why 🤷‍♂️

**Comment by knaaptime** [comment]
nope. still an issue. I have no idea how to solve this. The pip version is complete; the conda version is broken because bls.csv is missing

**Comment by knaaptime** [comment]
#322 partially addresses this by providing a fallback to the online version of the file, but it doesn't solve the root issue that the file is omitted during the conda-forge package build

**Comment by sjsrey** [comment]
Using geosnap 0.8.2, if the inflator is required (i.e., creating a community), there is a traceback about a missing dep:
```
ImportError: Missing optional dependency 'openpyxl'.  Use pip or conda to install openpyxl.

```

Installing that and restarting raises 
```
HTTPError: HTTP Error 403: Forbidden
```
when trying to access the bls data.

I'm wondering if it would be worth adding the inflator series inside the `adjust_inflation` function rather than having to hit bls?

**Comment by sjsrey** [comment]
It could be that:

https://github.com/spatialucr/geosnap/blob/master/setup.py#L78

needs to be updated to bring `io/bls_inflation.parquet` along for the ride?

**Comment by knaaptime** [comment]
resolved by #324
